transcom/mymove
# B 19705

## [19703](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A944626)
## [19705](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A944646)

## [Integration Review](https://github.com/transcom/mymove/pull/12954)

## Summary

19703 and 19705 were worked together due to the backend PPM objects being shared in some instances. 

These tasks are to remove the PPM pickupPostalCode, destinationPostalCode, secondaryPickupPostalCode, and secondaryDestinationPostalCode from the Customer, Office and Prime application. Test steps provided walk through the primary PPM pages to ensure functionality remains.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Customer PPM Tests:
1. Access the application as a customer
2. Create a new move
3. Create a new PPM Shipment for the move
4. Finalize the shipment
5. Finish and sign the move

SC Office PPM Tests:
1. Access the application
2. Sign is as a service counselor
3. Find the move you created
4. Approve the PPM you created
5. Submit move details 

TOO Office PPM Tests:
1. Sign into the application
2. Sign in as a TOO
3. Find the move you created
4. Edit the ppm details to ensure updating works

Prime PPM Tests: ( I used move code: 648TQB. Some of the moves don't have fully populated test data and don't work for some functions)
1. Access the application Prime Simulator
2. Open a move
3. Create a new PPM Shipment
4. Edit the PPM Shipment
5. Delete the PPM Shipment

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
